Content/RootScript/BaseClasses/Entities/Item.as

```diff
@@ -0,0 +1,174 @@
+namespace Item {
+	shared array<Item::Template@> templates;
+	shared array<Item@> instances;
+	shared void Register(Item::Template@&in template) {
```

Comment:
Please adhere to the style guide laid out in the readme.

Reply:
Variable names can be cleaned up as-needed.

Reply:
I have translated most of the variables into the desired naming convention.
May have missed a few though, we can fix that as we go.